connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/ClientE2ETestSuite.scala

```diff
@@ -220,41 +220,132 @@ class ClientE2ETestSuite extends RemoteSparkSession with SQLHelper {
     }
   }
 
+  test("writeTo with create") {
+    assume(IntegrationTestUtils.isCatalystTestJarAvailable)
+    withTable("testcat.myTableV2") {
+      spark.conf.set(
```

Comment:
can we use `withSQLConf` ?

Reply:
Or should we make this the default config for `SimpleSparkConnectService` startup, then we can skip this manual configuration

@zhenlineo WDYT?

Reply:
I removed it and change `RemoteSparkSession` to set config `spark.sql.catalog.testcat` as default.